### ISSUE TITLE
Showing the latest release next to the current branch might be confusing

### ISSUE THREAD

**Issue by bfred-it** [sindresorhus/refined-github#1198]
We might have to revert #1133

---

<img width="206" alt="master + 4.2.0" src="https://user-images.githubusercontent.com/1402241/37704998-88cac2f2-2d35-11e8-9085-8940ce22057c.png">

Is clear that we're on master and not _necessarily_ on 4.2.0?

---

<img width="182" alt="4.1.0 + 4.2.0" src="https://user-images.githubusercontent.com/1402241/37705412-e71db8e0-2d36-11e8-9057-b83e5d3f6834.png">

Where are we? 4.1.0 or 4.2.0?

---

In short, both pieces look like buttons, but one shows the current one and the other one is a link.

**Comment by sindresorhus** [comment]
I've been meaning to open an issue about this too. I find it very useful to quickly see the latest version, but we should find a better place to put the text. I think we should still keep the button though, just without the text.

**Comment by bfred-it** [comment]
Perhaps we should move the tag on the left. 

Or maybe move both _links_ away from the branch selector.

<img width="407" alt="links away from selector" src="https://user-images.githubusercontent.com/1402241/37705711-eb34cfbc-2d37-11e8-9528-1f45debe6721.png">

<img width="450" alt="show default branch name" src="https://user-images.githubusercontent.com/1402241/37705771-128eba00-2d38-11e8-9d4d-63daa346b3d5.png">

<img width="447" alt="different color" src="https://user-images.githubusercontent.com/1402241/37705837-456b5924-2d38-11e8-97d3-1bb04af10cde.png">

**Comment by SuperSandro2000** [comment]
Must agree this is very confusing at first.

**Comment by hkdobrev** [comment]
Why not use what GitHub is using on the their releases page and just say "Latest release" in a green box. Explicit is better than implicit. We can shorten to "<icon> Latest 1.9.3".

![image](https://user-images.githubusercontent.com/506129/37740817-c7cae63a-2d67-11e8-8e29-3af987508b3f.png)

https://github.com/sindresorhus/refined-github/releases

I like the all the branch-related things to be together like now. Still not sure about the left chevron arrow.

**Comment by SuperSandro2000** [comment]
I still think that it wouldn't be 100% clear if you are on that tag or branch.

**Comment by bfred-it** [comment]
@hkdobrev not a terrible idea, but that green label is used by GitHub to mark "**this** is the latest release", so it's not much clearer. If we could detect that our current view **is** the latest release (even on master) it'd be nice to turn it green.

I think my last mockup (grouped separately, blue links) is slightly more clear about them being links. It'd also drop the chevron you're not sure about.

**Comment by sindresorhus** [comment]
I like @bfred-it’s mock-up.

**Comment by bfred-it** [comment]
Related: the buttons should probably be moved away from here as well, perhaps to the right?

<a href="https://github.com/sindresorhus/refined-github/blob/history-handler-once-for-all/.travis.yml">
<img width="991" alt="screen shot 2018-03-22 at 13 13 10" src="https://user-images.githubusercontent.com/1402241/37752381-e28bc442-2dd2-11e8-8242-1f46ebc6d751.png"></a>

https://github.com/sindresorhus/refined-github/blob/history-handler-once-for-all/.travis.yml
